### PR TITLE
Ft resilience activities

### PR DESCRIPTION
**Pull request by BrendaNamutebi**
PR Checklist (Make sure these options are checked before raising the PR)

- [x] - Latest `master` branch merged
- [x] - Descriptive PR title (can be used in release notes)

#### Summary of Changes (What does this PR do?)
- Mapped resilience activities to each own's specific content

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Jira Ticket(s) Number
- [SA-50]

## Screenshots/Videos
image
image

### CODE REVIEW

**Comment by FaithDaka** [comment]
@BrendaNamutebi You might want to clean your latest commits. Most of the files can't be found so I don't think you pulled the latest changes before you made your updates today.
Resource: https://mattstauffer.com/blog/how-to-merge-only-specific-commits-from-a-pull-request/

File: src/pages/ResilienceActivities/Activities/Journaling.jsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+  const handlePageChange = (page) => {
+    setCurrentPage(page);
+  };
+
+
+  console.log(data, '>>>>>>>>>>>>>>>>>>>>>>');
+
+
+  return (
```
Comment (KingMike100) [inline src/pages/ResilienceActivities/Activities/Journaling.jsx:124]: @BrendaNamutebi Find a way to reload this page whenever a "grateful/note" is saved to imporve the user experience, at the moment, it requires a hard refresh to the server to update the page.

Comment (KingMike100) [reply src/pages/ResilienceActivities/Activities/Journaling.jsx:124]: There's also a bit of styling errors happening as the save button is not flexible as the new notes are added

Comment (KingMike100) [reply src/pages/ResilienceActivities/Activities/Journaling.jsx:124]: When you select the notebooks, they're loaded well in a listing but once you tap on an individual one to display it. Its not displayed, only titles are displayed.

File: src/pages/ResilienceActivities/Activities/OneServiceGoal.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            </div>
+            <hr style={{ width: 325, marginTop: '0px' }} />
+            <div>
+              <p style={{ fontSize: '11px', marginLeft: '6px' }}>
+                Doing acts of service which can be described as doing something for someone else
+                which you know they would like can have a profound positive impact on mood and
+                well-being. Pick one service goal to complete in the next few days. Helping someone
+                or a group can benefit to your well-being!
+              </p>
+              <h6 style={{ fontSize: '13px', marginLeft: '25px', marginTop: '3px' }}>Tips:</h6>
```
Comment (KingMike100) [inline src/pages/ResilienceActivities/Activities/OneServiceGoal.jsx:41]: can you define these under classesin a separate file for readability, not a good practice having one tag(h6) with different styles

**Comment by KingMike100** [comment]
Thanks @BrendaNamutebi for this attempt, its a good start with a couple of changes needed especially on the journaling

File: src/pages/ResilienceActivities/Activities/Journaling.jsx (hunk 1 above)
Comment (BrendaNamutebi) [reply src/pages/ResilienceActivities/Activities/Journaling.jsx:124]: Still handling an endpoint for getting a single journal

**Comment by khalifan-kfan** [comment]
@BrendaNamutebi, handled the journaling, functionality, and some UI fixes. Left you one UI fix to resolve

**Pull request merged**